### PR TITLE
Instruction and tool for OpenVPN adapter configuration.

### PR DESCRIPTION
**Pull request by ababo**
This change contains:
1. README.md for dappvpn configuration.
2. Installation tool which creates a new product in the database and updates a given dappvpn configuration.

Resolves BV-266.

### CODE REVIEW

File: tool/dappinst/main.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+	"github.com/AlekSi/pointer"
+	"gopkg.in/reform.v1"
+
+	"github.com/privatix/dappctrl/data"
+	"github.com/privatix/dappctrl/util"
+)
+
+const (
+	json = `{
```
Comment (sofabeat) [inline tool/dappinst/main.go]: Let's create two different jsons. We need them.

File: tool/dappinst/main.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+		ID:            util.NewUUID(),
+		Name:          "OpenVPN server",
+		OfferTplID:    pointer.ToString(offer.ID),
+		OfferAccessID: pointer.ToString(access.ID),
+		UsageRepType:  data.ProductUsageTotal,
+		IsServer:      true,
+		Salt:          salt,
+		Password:      passwordHash,
+		ClientIdent:   data.ClientIdentByChannelID,
+		Config:        json,
```
Comment (sofabeat) [inline tool/dappinst/main.go]: `Config` should be empty. I've confused you.

**Comment by ababo** [comment]
@sofabeat updated

**Comment by ababo** [comment]
@kelewind updated

**Pull request merged**